### ISSUE TITLE
README.md has not been reflected in what was deployed to Maven Central repository.

### ISSUE THREAD

**Issue by yoshikawaa** [terasolunaorg/terasoluna-gfw#728]
## Description
In README.md
```xml
Irrespective of above two ways, first define the repositories in the pom file.

(The below is required in the both the type of settings.)

<repositories>
    <repository>
        <releases>
            <enabled>true</enabled>
        </releases>
        <snapshots>
            <enabled>false</enabled>
        </snapshots>
        <id>terasoluna-gfw-releases</id>
        <url>http://repo.terasoluna.org/nexus/content/repositories/terasoluna-gfw-releases/</url>
    </repository>
</repositories>
```
5.3.0 and later, repositories setting is unnecessary, because `terasoluna-gfw` is already deployed to Maven Central repository.

## Possible Solutions
Remove the description.

## Affects Version/s
- 5.3.X.RELEASE and later

## Fix Version/s
- [x] 5.5.0 (master)
- [x] 5.4.1 (5.4.x)
- [x] 5.3.2 (5.3.x)

**Comment by kazuki43zoo** [comment]
@yoshikawaa Good catch 👍  But it might be better to consider user using old version(before 5.2.x).

**Comment by yoshikawaa** [comment]
@kazuki43zoo 
It may be kind to explain both [5.3.0 and later] or [earlier].
However, since there is a README for each version, there may be opinion that it should explain appropriate things for each version.
We will consider how this should be, Thanks.

@itouyun 
Let's consider based on the above.

**Comment by itouyun** [comment]
Thank you.

- Plan A
  - explain both [5.3.0 and later] or [earlier].
- Plan B
  - fix only 5.3.x and master, these explain only Central repository.

I think that Plan B is better because user uses user's version of `terasoluna-gfw`.

**Comment by yoshikawaa** [comment]
@btiwahorisu Please PR to 5.4.x branch.
